libs/vscode-ui/components/src/lib/autocomplete/autocomplete.component.html

```diff
@@ -1,3 +1,38 @@
-<datalist id="{{ field.name }}_enum">
-  <option *ngFor="let value of field.enum" [value]="value"></option>
-</datalist>
+<div class="description">
+  {{ field.description }}
+</div>
+
+<div class="modified-indicator"></div>
+
+<div class="control">
+  <div class="autocomplete-container">
+    <input
+      #textInput
+      [formControl]="control"
+      class="text-input"
+      type="text"
+      role="textbox"
+      [required]="field.required"
```

Comment:
Do we need this binding? What does it give us in terms of UX? If it doesn't add value remove it since removing as many bindings as we can improves our time to initial render

Reply:
No we don't, I'll remove it.

Reply:
`required` binding removed